Makefile

```diff
@@ -241,6 +271,16 @@ compilepy3:
 	@sudo rabbitmqctl reset
 	@sudo rabbitmqctl start_app
 
+.PHONY: .cleancoverage
+.cleancoverage:
+	@echo "==================== cleancoverage ===================="
+	@echo "Removing all coverage results directories"
+	@echo
+	rm -rf .coverage \
+	 .coverage-unit .coverage-unit-* \
```

Comment:
Would be safer to quote every parameter passed to `rm -rf` so it doesn't accidentally become `.coverage-unit- *` or similar in the future.